basics_of_logging_in_sst-core/Simplified_SST_Logging.md

```diff
@@ -0,0 +1,329 @@
+# SIMPLIFIED SST LOGGING
+
+## INTRODUCTION
+
+Logging is quite possibly one of the most important features built into an application.  It is useful not only for debugging but for analyzing what happened when problems occur after deployment.
+
+To support logging, the Structural Simulation Toolkit (SST) includes a logging class that provides a standardized way to format and output logging messages.  Like most, it allows the developer to specify a formatted text string that will be printed before each log message, the logging level, and has the option to send log messages to the terminal or log file.
+
+However, for those developers first coming to SST the interface can be slightly confusing.  This documentation is intended to serve as a brief introduction to the SST logging class, explain how its features map to those from other logging libraries, and provide ways to simplify the interface to something a developer may be more familiar with.
+
+## A BRIEF INTRODUCTION TO LOGGING
+
+As stated by Colin Eberhardt, "Logging is the process of recording application actions and state to a secondary interface." (Eberhardt, 2014)  Recording application actions usually takes the form of a printed message and the secondary interface can be a local file, network storage, or even the terminal.
+
+According to the Syslog Protocol (Gerhards, 2009), logging messages can be categorized as:
```

Comment:
I'm not sure syslog is the best model to use for logging in SST.  It was really intended to monitor running systems, not individual applications. Since each element gets to specify their own levels based off of the complexity of the code, it would be difficult to map things into these discrete buckets.  

In general, this is advocating for something that is very different than what is currently implemented in the elements, but presented like it is standardized.  That's fine if users want to use a model like this, but I worry about presenting it in a beginning tutorial in a way that may actually confuse the user when they use the main SST elements repository.